Services/Concurrency/RateLimiting.cs

```diff
@@ -58,6 +60,50 @@ public class RateLimiting : IRateLimiting
             log.Info("Rate limiting started. This message should appear only once in the logs.");
         }
 
+        public Models.Simulation.SimulationRateLimits GetRateLimits()
```

Comment:
Naming seems inconsistent for GetRateLimits and SetCounters. Can we rename these to be consistent ?